src/wolfsftp.c

```diff
@@ -1007,6 +1010,8 @@ int wolfSSH_SFTP_read(WOLFSSH* ssh)
     int maxSz, ret = WS_SUCCESS;
     WS_SFTP_RECV_STATE* state = NULL;
 
+    WLOG(WS_LOG_DEBUG, "Entering wolfSSH_SFTP_read()");
```

Comment:
please change to be WS_LOG_SFTP log type. Trying to keep all SFTP debugging unique so a quick filter on the connections debugging output using WS_LOG_SFTP will show all SFTP related debug messages.

Reply:
What I'd like to see is the logging have a level (WARN, TRACE, DEBUG, etc) and a domain (KEX, SFTP, SCP, etc). Right now it is a bad mashing of the two and the pattern not followed at all.